### PR TITLE
 fix #3 create packaging/installation setup for Mac

### LINKED ISSUE #3: create packaging/installation setup for Mac

### PR DESCRIPTION
**Pull request by shubhamvashisht**
To build for mac, Run --->   npm run dist-mac

### CODE REVIEW

**Comment by piyush0** [comment]
screen shot 2017-06-04 at 12 09 54 am

Getting this error. Please check

**Comment by shubhamvashisht** [comment]
I forgot to add,
first you will have to install [Electron Builder](https://www.npmjs.com/package/electron-builder]) to build the project.
1. Run--->   `npm install electron-builder --save-dev` to install electron builder.
then
2. Run--->  `npm run dist-mac` to build.

**Comment by piyush0** [comment]
👍 
@championswimmer Reviewed this. Ready to merge.

File: package.json [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
   "license": "ISC",
   "dependencies": {
-    "electron": "^1.6.6",
     "mongodb": "^2.2.28",
     "path": "^0.12.7",
     "url": "^0.11.0"
+  },
+  "devDependencies": {
+  	"electron": "^1.6.6",
+    "electron-builder": "^18.3.5"
```
Comment (championswimmer) [inline package.json]: indendation

File: package.json [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
   "main": "./main.js",
   "scripts": {
+  	"postinstall": "install-app-deps",
+  	"pack": "build --dir",
+    "dist-mac": "build --mac",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "electron ."
   },
+  "build": {
+    "appId": "coding-blocks.Snipper",
```
Comment (championswimmer) [inline package.json:14]: can an App Icon be also added (use a dummy square icon of CB for now, we will design a icon separately later)

**Review by championswimmer — CHANGES_REQUESTED** [review]

**Comment by shubhamvashisht** [comment]
1. Icon for mac package added.
2. Indentation fixed.

**Pull request merged**